cypress/cypress/integration/recommended-questions.spec.ts

```diff
@@ -63,9 +68,30 @@ describe("Recommended questions", () => {
         recommendedQuestions: "Are you fun at parties?",
       }),
     });
-    cy.get("[data-cy=topic-0]").find(".topic-selected");
-    cy.get("[data-cy=topic-2]").trigger("mouseover").click();
-    cy.get("[data-cy=topic-2]").find(".topic-selected");
+    cy.get("[data-cy=topics]")
```

Comment:
I'm not sure what this is supposed to test?

If there's any alternative, we should avoid testing css properties (very fragile tests and hard to understand for programmers reading it later)

So the two alternatives are:

 - test something about what the user sees (e.g. test the value property of a dropdown)
 or
 - use `data-test` attributes and then test the values

see https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
https://github.com/mentorpal/mentor-client/pull/87/commits/4357d304bb789efd2a03a73910bf02aabbe849e3